src/config/rollup.config.js

```diff
@@ -146,13 +159,15 @@ module.exports = {
     nodeResolve({
       preferBuiltins: isNode,
       mainFields: ['module', 'main', 'jsnext', 'browser'],
+      extensions,
     }),
     commonjs({include: 'node_modules/**'}),
     json(),
     rollupBabel({
       presets: babelPresets,
       babelrc: !useBuiltinConfig,
-      runtimeHelpers: useBuiltinConfig,
+      runtimeHelpers: hasAnyDep('@babel/runtime'),
```

Comment:
Let's use `hasDep` rather than `hasAnyDep` (looks like we'll need to create that). If they don't have it in their regular deps then it may not work for consumers.

Reply:
changed to `hasDep`